script.js

```diff
@@ -1,4 +1,108 @@
+var GAME_STATE_DICE_ROLL = "GAME_STATE_DICE_ROLL";
+var GAME_STATE_CHOOSE_DICE_ORDER = "GAME_STATE_CHOOSE_DICE_ORDER";
+var GAME_STATE_CHOOSE_COMPARE_SCORES = "GAME_STATE_COMPARE_SCORES";
+var gameState = GAME_STATE_DICE_ROLL;
+
+var currentPlayerRolls = [];
+
+var currentPlayer = 1;
+var allPlayersScore = [];
+
+var playerWins = [0, 0, 0];
+
+var rollDice = function () {
+  var randomDecimal = Math.random() * 6;
+  var randomInteger = Math.floor(randomDecimal) + 1;
+  return randomInteger;
+};
+
+var rollDiceForPlayer = function () {
+  var counter = 0;
+  while (counter < 2) {
+    currentPlayerRolls.push(rollDice());
+    counter = counter + 1;
+  }
+  return `Welcome, Player ${currentPlayer}. <br><br>You rolled:<br>Dice 1: ${currentPlayerRolls[0]}| Dice 2: ${currentPlayerRolls[1]}.<br><br>Now, please input '1' or '2'`;
+};
+
+var getPlayerScore = function (playerInput) {
+  var playerScore;
+  if (playerInput == 1) {
+    playerScore = Number(
+      String(currentPlayerRolls[0]) + String(currentPlayerRolls[1])
+    );
+    allPlayersScore.push(playerScore);
+    currentPlayerRolls = [];
+    return `Player ${currentPlayer}, your chosen value is : ${playerScore}`;
```

Comment:
This seems to be a common piece of code between the two `if` conditions for `if (playerInput == 1)` and `if (playerInput == 2)`, is it possible to combine the two as below:
```
var getPlayerScore = function (playerInput) {
  var playerScore;
  if (playerInput == 1) {
    playerScore = Number(
      String(currentPlayerRolls[0]) + String(currentPlayerRolls[1])
    );
  } else if (playerInput == 2) {
    playerScore = Number(
      String(currentPlayerRolls[1]) + String(currentPlayerRolls[0])
    );
   }
   allPlayersScore.push(playerScore);
   currentPlayerRolls = [];
   return `Player ${currentPlayer}, your chosen value is : ${playerScore}`;
};